CONTRIBUTING.md

```diff
@@ -4,15 +4,46 @@ This document provides information about contributing to the [Keptn documentatio
 This documentation is authored with [markdown](https://www.markdownguide.org/basic-syntax/)
 and rendered using the [Hugo static website generator framework](http://gohugo.io).
 
+We welcome and encourage contributions of all levels.
+You can make modifications using the github editor;
+this works well for small modifications but, if you are making significant changes, you may find it better to fork and clone the repository and make changes using the text editor or IDE of your choice
+You can run the Hugo formatter locally to check the rendered documentation
+and then push your changes to the repository as a PR.
+
+If you need help getting started, feel free to ask for help on the #keptn-help Slack channel.
+We were all new to this once and are happy to help you!
+
+## Guidelines for contributing
+
+* Always fork the repository then clone that fork to your local system rather than clone master directly.
+Keptn, like most open source projects, severely restricts who can push changes directly to the *master* branch to protect the integrity of the repository.
+* Keep your language clean and crisp.
+* Smaller PR's are easier to review and so generally get processed more quickly;
+when possible, chunk your work into smallish PR's.
+However, we recognize that documentation work sometimes requires larger PRs, such as when writing a whole new section or reorganizing existing files.
+* You may want to squash your commits before creating the final PR, to avoid conflicting commits.
```

Comment:
```suggestion
* You may want to squash your commits before creating the final PR, to avoid conflicting commits.
   It is **not mandatory**, the maintainers can always squash commits during the merge.
```